api/v1alpha1/ndbserver_types.go

```diff
@@ -44,8 +44,8 @@ type ReconcileCounter struct {
 	Database int `json:"database"`
 }
 
-//+kubebuilder:object:root=true
-//+kubebuilder:subresource:status
+// +kubebuilder:object:root=true
```

Comment:
just for my knowledge, what does this do?

Reply:
This marker tells the Operator SDK's underlying Kubebuilder to treat this golang type struct as a Kubernetes Kind/resource. This marker generates the implementation of the Object interface that all Kubernetes `Kinds` are supposed to implement.

More here:
1. https://book.kubebuilder.io/cronjob-tutorial/new-api
2. https://book.kubebuilder.io/reference/markers/object